### ISSUE TITLE
Problem using greater_equals and greater_than

### ISSUE THREAD

**Issue by k0pernicus** [dgraph-io/tutorial#10]
When I am using `ge` and `gt` (or `le` and `lt`), nothing changed:

```
{
  michael_friends_sorted(func: allofterms(name, "Michael")) {
    name
    age
    friend @filter(ge(age, 35)) (orderasc: age) {
        name@.
        age
    }
  }
}
```

Here, the response is ok:
```
{
  "michael_friends_sorted": [
    {
      "age": 39,
      "friend": [
        {
          "age": 35,
          "name@.": "Artyom"
        },
        {
          "age": 35,
          "name@.": "Amit"
        },
        {
          "age": 55,
          "name@.": "Sarah"
        }
      ],
      "name": "Michael"
    }
  ]
}
```

But, when I am using `gt`:
```
{
  michael_friends_sorted(func: allofterms(name, "Michael")) {
    name
    age
    friend @filter(gt(age, 35)) (orderasc: age) {
        name@.
        age
    }
  }
}
```

The response is unchanged, and it give me a graph than contains friends who have 35 years:
```
{
  "michael_friends_sorted": [
    {
      "age": 39,
      "friend": [
        {
          "age": 35,
          "name@.": "Artyom"
        },
        {
          "age": 35,
          "name@.": "Amit"
        },
        {
          "age": 55,
          "name@.": "Sarah"
        }
      ],
      "name": "Michael"
    }
  ]
}
```

**Comment by sboorlagadda** [comment]
This [issue](https://github.com/dgraph-io/dgraph/issues/2085) is fixed in 1.0.3

**Comment by k0pernicus** [comment]
Thanks @sboorlagadda  :-)
